### ISSUE TITLE
Add panel layout options

### ISSUE THREAD

**Issue by oevans** [Esri/Compare-Maps-Template#4]
It would be great to have a configuration option to **choose between side-by-side or stacked layout** of map panels.  If your maps have a horizontal aspect ratio (e.g., the state of Tennessee or Maryland) the side-by-side layout is not ideal, especially with more than two maps.

If comparing different sized areas, it would be useful to be able to **specify individual panel widths** (or heights in stacked configuration).  Preferably using percentages in the configuration panel or interactively using a handle on the panel separators.

I could also see a **diagonal panel split** working well for to maximize the space when comparing two square-shaped areas of interest, say for Colorado or Australia.

**Comment by kellyhutchins** [comment]
@oevans  I agree that this would be useful and can look at it for a future enhancement.
